snapcraft/internal/lxd.py

```diff
@@ -139,6 +141,56 @@ def _setup_project(self):
         self._push_file(tar_filename, dst)
         self._container_run(['tar', 'xvf', os.path.basename(tar_filename)])
 
+    def _inject_snapcraft(self):
+        if common.is_snap():
+            # Because of https://bugs.launchpad.net/snappy/+bug/1628289
+            self._container_run(['apt-get', 'install', 'squashfuse', '-y'])
+
+            # Push core snap into container
+            self._inject_snap('core', '99T7MUlRhtI3U0QFgl5mXXESAiSwt776')
+            snapcraft_rev = os.environ.get('SNAP_REVISION')
+            self._inject_snap('snapcraft', 'vMTKRaLjnOJQetI78HjntT37VuoyssFE',
+                              rev=snapcraft_rev, classic=True)
+        else:
+            self._container_run(['apt-get', 'install', 'snapcraft', '-y'])
+
+    def _inject_snap(self, name, id, *, rev=None, classic=False):
+        if not rev:
+            rev = check_output(['readlink',
+                                os.path.join(os.path.sep,
+                                             'var', 'snap',
+                                             name, 'current')]).decode()[:-1]
+        key = \
```

Comment:
We prefer parentheses instead of \.
And by *we*, I mean *I*, because I'm not sure if we have discussed this nit before. Do you prefer \ for some reason?